src/torrent.cpp

```diff
@@ -121,9 +124,12 @@ namespace libtorrent
 		}
 		return ret;
 	}
-
 	constexpr int default_piece_priority = 4;
 
+	// seed rank priority duration which need to match ratio type which is used at
+	// seed rank calculation to prevent runtime conversion and constructions
+	constexpr seconds32 seed_rank_priority_duration = seconds32(30 * 60);
```

Comment:
I will move it back to inline because the performance gain is not worth the extra 5 lines. I gues compiler is smart enoght to create this constexpr out of an inline class construct.